.gitignore

```diff
@@ -45,4 +45,5 @@ buildNumber.properties
 .classpath
 .project
 .settings/
-.vscode/
\ No newline at end of file
+.vscode/
+coffeeshop-chart/charts/keda-1.0.0.tgz
```

Comment:
you could make this simply `coffeeshop-chart/charts/`
(I probably should have included that in #6)

Reply:
Done